src/ostorlab/agent/message/proto/v3/use/device/device.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto2";
+
+package ostorlab.agent.message.proto.v3.use.device;
+
+
+message LoginPassword {
+  optional string username = 1;
+  optional string password = 2;
+}
+
+message SSHKey {
+  optional string private_key = 1;
+  optional string public_key = 2;
+}
+
+message Credentials {
+  oneof credentials {
+    LoginPassword login_password = 1;
+    SSHKey ssh_key = 2;
+  }
+}
+
+message DeviceRelay {
+  optional string name = 1;
+  optional string address = 2;
+  optional sint32 port = 3;
+}
+
+message Duration {
+  optional int32 nanos = 1;
+  optional int64 seconds = 2;
+}
+
+message Message {
```

Comment:
The message is supposed to contain multiple devices, no a single device

Reply:
Each device will have his own message.